.github/workflows/main_ci.yml

```diff
@@ -0,0 +1,38 @@
+name: Main CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install libsndfile
+      run: |
+        sudo apt-get install libsndfile1
```

Comment:
Is this related to librosa lib?

Reply:
Yes i added it because of a missing sound lib